<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TtRssAccountDetails</class>
 <widget class="QWidget" name="TtRssAccountDetails">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>432</width>
    <height>396</height>
   </rect>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="10" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>408</width>
       <height>30</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="m_lblTitle">
       <property name="text">
        <string>URL</string>
       </property>
       <property name="buddy">
        <cstring>m_txtUrl</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="LineEditWithStatus" name="m_txtUrl" native="true"/>
     </item>
    </layout>
   </item>
   <item row="2" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Only download newest X articles per feed</string>
       </property>
       <property name="buddy">
        <cstring>m_spinLimitMessages</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="MessageCountSpinBox" name="m_spinLimitMessages">
       <property name="maximumSize">
        <size>
         <width>140</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QCheckBox" name="m_checkDownloadOnlyUnreadMessages">
     <property name="text">
      <string>Download unread articles only</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QCheckBox" name="m_cbNewAlgorithm">
     <property name="text">
      <string>Intelligent synchronization algorithm</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QCheckBox" name="m_checkServerSideUpdate">
     <property name="text">
      <string>Force execution of server-side feeds update</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="2">
    <widget class="QGroupBox" name="m_gbAuthentication">
     <property name="toolTip">
      <string>Some feeds require authentication, including GMail feeds. BASIC, NTLM-2 and DIGEST-MD5 authentication schemes are supported.</string>
     </property>
     <property name="title">
      <string>Authentication</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Username</string>
        </property>
        <property name="buddy">
         <cstring>m_txtUsername</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="LineEditWithStatus" name="m_txtUsername" native="true"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Password</string>
        </property>
        <property name="buddy">
         <cstring>m_txtPassword</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="LineEditWithStatus" name="m_txtPassword" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="8" column="0" colspan="2">
    <widget class="QGroupBox" name="m_gbHttpAuthentication">
     <property name="toolTip">
      <string>Some feeds require authentication, including GMail feeds. BASIC, NTLM-2 and DIGEST-MD5 authentication schemes are supported.</string>
     </property>
     <property name="title">
      <string>Requires HTTP authentication</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QFormLayout" name="formLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Username</string>
        </property>
        <property name="buddy">
         <cstring>m_txtHttpUsername</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="LineEditWithStatus" name="m_txtHttpUsername" native="true"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Password</string>
        </property>
        <property name="buddy">
         <cstring>m_txtHttpPassword</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="LineEditWithStatus" name="m_txtHttpPassword" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="9" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="m_btnTestSetup">
       <property name="text">
        <string>&amp;Test setup</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="LabelWithStatus" name="m_lblTestResult" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="HelpSpoiler" name="m_lblNewAlgorithm" native="true"/>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="HelpSpoiler" name="m_lblServerSideUpdateInformation" native="true"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LabelWithStatus</class>
   <extends>QWidget</extends>
   <header>labelwithstatus.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>LineEditWithStatus</class>
   <extends>QWidget</extends>
   <header>lineeditwithstatus.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MessageCountSpinBox</class>
   <extends>QSpinBox</extends>
   <header>messagecountspinbox.h</header>
  </customwidget>
  <customwidget>
   <class>HelpSpoiler</class>
   <extends>QWidget</extends>
   <header>helpspoiler.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
